feat(js): Add js tracing without performance docs (#7421)
getsentry/sentry-docs: src/platform-includes/distributed-tracing/how-to-use/node.mdx

If you use the version `7.58.0` or above of our Node SDK, distributed tracing just works out of the box if you've set up the HTTP integration.

```js
Sentry.init({
  dsn: "___PUBLIC_DSN___",
  integrations: [new Sentry.Integrations.Http({ tracing: true })],
});
```

> If you're using express or an express-compatible framework, you also need to use the Sentry middleware:

Reviewer: ```suggestion
If you're using express or an express-compatible framework, you'll also need to use Sentry middleware:
```

```js
const app = express();

// RequestHandler creates a separate execution context, so that all
// transactions/spans/breadcrumbs/tags are isolated across requests
app.use(Sentry.Handlers.requestHandler());
// TracingHandler creates a trace for every incoming request
app.use(Sentry.Handlers.tracingHandler());
```

> If you however use version `7.57.x` or below, you need to have our <PlatformLink to="/performance/">performance monitoring feature enabled</PlatformLink> for distributed tracing to work.

Reviewer: ```suggestion
If you're using version `7.57.x` or below, you'll need to have our <PlatformLink to="/performance/">performance monitoring feature enabled</PlatformLink> in order for distributed tracing to work.
```
